lms/djangoapps/support/static/support/jsx/entitlements/data/api/client.js

```diff
@@ -0,0 +1,53 @@
+import 'whatwg-fetch';
+import Cookies from 'js-cookie';
+
+import { entitlementList } from './endpoints';
+
+const headers = {
+  Accept: 'application/json',
+  'Content-Type': 'application/json',
+  'X-CSRFToken': Cookies.get('csrftoken'),
+}
+
+export function requestEntitlements({usernameOrEmail}) {
```

Comment:
It would be clearer for the `usernameOrEmail` / `user` / `email` parameters in the three different functions to all have the one name, since they represent the same piece of information in each of the three requests.